kie-server-parent/kie-server-controller/kie-server-controller-impl/src/main/java/org/kie/server/controller/impl/service/RuleCapabilitiesServiceImpl.java

```diff
@@ -43,10 +45,13 @@
     public void scanNow(final ContainerSpecKey containerSpecKey) {
         ServerTemplate serverTemplate = templateStorage.load(containerSpecKey.getServerTemplateKey().getId());
         if (serverTemplate == null) {
-            throw new RuntimeException("No server template found for id " + containerSpecKey.getServerTemplateKey().getId());
+            throw new KieServerControllerNotFoundException("No server template found for id " + containerSpecKey.getServerTemplateKey().getId());
```

Comment:
exception class name sounds bit weird - it seems to indicate that kie server controller was not found... maybe rename it a bit to be more meaningful?

Reply:
Yeah, perhaps it should be treated as an illegal argument instead.
Renamed to _KieServerControllerIllegalArgumentException_